server/ai_http.go

```diff
@@ -139,8 +155,85 @@ func (h *lphttp) StartLiveVideoToVideo() http.Handler {
 			}
 		}()
 
-		// TODO subscribe to the subscribeUrl for output monitoring
+		// TODO: Use params temporarily to pass the host from gateway
+		host := req.PublishUrl
+		pub, err := common.AppendHostname(pubUrl, host)
+		if err != nil {
+			respondWithError(w, fmt.Errorf("invalid publish URL: %w", err).Error(), http.StatusBadRequest)
+			return
+		}
+		sub, err := common.AppendHostname(subUrl, host)
+		if err != nil {
+			respondWithError(w, fmt.Errorf("invalid subscribe URL: %w", err).Error(), http.StatusBadRequest)
+			return
+		}
+		control, err := common.AppendHostname(controlUrl+"-control", host)
+		if err != nil {
+			respondWithError(w, fmt.Errorf("invalid control URL: %w", err).Error(), http.StatusBadRequest)
+			return
+		}
+
+		// Prepare request to worker
+		workerReq := worker.LiveVideoToVideoParams{
+			ModelId:      req.ModelId,
+			PublishUrl:   sub, // SubscribeUrl is the publish url for the worker
```

Comment:
`// SubscribeUrl is the publish url for the worker` => I'm not sure I understand this comment. Could you rephrase it?

Reply:
Updated comment to `Subscribe and Publish urls are swapped for the worker`